programmers/2일차/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+    let answer = "";
+    let splitS = s.split("");
+    if (splitS.length % 2 !== 0 ) {
+            answer = splitS[parseInt(splitS.length / 2)];
+            console.log(answer);
+    }
+    else {
+        answer = splitS[parseInt(splitS.length / 2)-1]+splitS[parseInt(splitS.length / 2)];
+    }
+    return answer;
+}
```

Comment:
```suggestion
function solution(s) {
    return s.substr(Math.ceil(s.length / 2) - 1, s.length % 2 === 0 ? 2 : 1);
}
```
이렇게  substr메서드와  삼항 연산자를 같이 쓰면  간결하게 해결할 수도 있답니다  
substr() 이나  slice() 같은 메서드는 정말 유용하고 자주 사용하는 문자열 메서드이기 때문에 한번 공부해보시는 걸 추천드립니다